src/include/uapi/ipc.h

```diff
@@ -259,7 +269,7 @@ struct sof_ipc_dai_ssp_params {
 	uint32_t bclk_keep_active;
 	uint32_t fs_keep_active;
 
-	//uint32_t quirks; // FIXME: is 32 bits enough ?
+	uint32_t quirks; // FIXME: is 32 bits enough ?
 
```

Comment:
since we are planning an IPC change, I was also thinking of adding a frame_width to program the pulse duration in DSP_A or DSP_B modes. This is a deviation from the regular ASoC modes where the pulse is assumed to be one but this is something that can be needed in specific cases

Reply:
Do you mean adding frame_width QUIRK macro (6 bits in length) to program SFRMWDTH field in SSPSP register?

Reply:
I've added SOF_DAI_INTEL_SSP_QUIRK_SFRMWDTH macro and I've set frame_len in SOF_DAI_FMT_DSP_A nad SOF_DAI_FMT_DSP_B mode (default value is 1). Is that what you were thinking about?

Reply:
Oh, and I forgot, split the patch in two commits so that the IPC change can be applied as is to the kernel part (the IPC needs to come first to avoid breaking git bisect)

Reply:
Oh, and I forgot, split the patch in two commits so that the IPC change can be applied as is to the kernel part (the IPC needs to come first to avoid breaking git bisect)